plugin.py

```
@@ -927,6 +975,47 @@ def _db_stats() -> str:
+    """
+    Database cleanup - deletes ALL episodes and episode relations.
+    Issue #556 - Quick database cleanup for development/debugging.
+
+    Replicates these SQL commands in Django ORM:
+        DELETE FROM vod_episode;
+        DELETE FROM vod_m3uepisoderelation;
+
+    Returns:
+        Formatted string with operation results
+    """
+    from django.db import transaction
```

Ruff (0.14.3)

Redefinition of unused `transaction` from line 40

Remove definition: `transaction`

(F811)